resources/qml/PrivacyScreen.qml

```diff
@@ -76,6 +81,11 @@ Item {
                     visible: false
                 }
 
+                PropertyChanges {
+                    target: Nheko
+                    privacyScreenActive: false
```

Comment:
I think you want instead set a property on the privacy screen instead of a global one, so that different screens don't cause images to randomly blur or not. And then have the image find the closest privacy screen in its parents or so (by giving them the same id or so?).

Reply:
You can test that by using multiple windows

Reply:
Yeah, that is a good point. I'll look into it